e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml

```diff
@@ -50,6 +46,47 @@ spec:
               value: "30100"
             - name: CLIENT_PORT_MAX
               value: "30105"
+            - name: POD_NAME
```

Comment:
There are a lot of environment variables that are not used anymore. For example, PMM_USER, CLIENT_PORT_MAX, etc

Reply:
it was discussed earlier that we will clean up old code in separate review or maybe in future release

Reply:
Well, _that_ was a separate review 😆